t/APISIX.pm

```diff
@@ -495,8 +495,6 @@ _EOC_
 
     # The new directive is introduced here to modify the schema
     # before apisix validate in require("apisix")
-    # Todo: merge extra_init_by_lua_start and extra_init_by_lua
```

Comment:
We need to change extra_init_by_lua so we don't need a separate extra_init_by_lua_start.
Otherwise the CI will fail: https://github.com/apache/apisix/actions/runs/3404934882/

Reply:
Do you mean the `extra_init_by_lua` function should include the extra_init_by_lua_start module?
This commit is my one attempt.